frontend/src/leaderboard.js

```diff
@@ -0,0 +1,31 @@
+const ACCOUNTS_URL = "http://localhost:3000/accounts";
+document.addEventListener('DOMContentLoaded', () => {
+  main();
+});
+function main() {
+  fetchUsers();
+}
+function fetchUsers() {
+  fetch(ACCOUNTS_URL)
+  .then(res => res.json())
+  .then(createAccounts);
+}
+function createAccounts(accounts) {
+  for (const account of accounts) {
+    console.log(account);
+    displayAccount(createAccount(account));
+  }
+}
+function displayAccount(accountDiv) {
+  const leaderboardDiv = document.getElementById('leaderboard-window');
+  leaderboardDiv.append(accountDiv);
+}
+function createAccount(account) {
+  const accountDiv = document.createElement('div');
+  const accountH2 = document.createElement('h3');
```

Comment:
probably adjust the name to match what type of H_ you go with

Reply:
oh good catch thanks.